### ISSUE TITLE
Amazfit Bip Lite No GPS signal received from Phone

### ISSUE THREAD

**Issue by rbk0** [Freeyourgadget/Gadgetbridge#1691]
#### Before reporting a bug, please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [x] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

### I got Gadgetbridge from:
* [x] F-Droid
* [ ] I built it myself from source code (specify tag / commit)

#### Your wearable device is:
Amazfit Bip Lite
Model: A1915
Firmware-Version: V1.1.6.32

#### Your android version is:
Oxygen OS:9.0.9
#### Your Gadgetbridge version is:
0.37.0

The Amazfit Bip Lite does not receive any GPS Signal.  Started different activities. Everytime same results.
Got my device few days ago and MiFit updated to the currsnt firmware.

Anyone has experience with older/other Firmware
versions?

Logs:
[huamidebug_20191018-120826.log](https://github.com/Freeyourgadget/Gadgetbridge/files/3743502/huamidebug_20191018-120826.log)

**Comment by IzzySoft** [comment]
You are aware that the Bip Lite doesn't have GPS? Unless you have the "normal" Bip which was "accidentally" named lite (just for confusion), that is.

**Comment by ashimokawa** [comment]
Mi fit sends the GPS data from the phone to the watch when using the bip lite, gadgetbridge does not support that yet

**Comment by rbk0** [comment]
Yeah. As mentioned in the title its about Amazfit Bip Lite not receiving any GPS information from the phone. Just forgot to mention it again afterwards.

Normally it should receive GPS information from a connected Phone. Just read in the device request  that it was nit implemented yet. Gonna try to get my hands on it and implement it myself.

Idea is to analyze the BT communcation from MiFit and adapt for GB. Anybody has experience with a better approach?

**Comment by ashimokawa** [comment]
@rbk0

No this is how it is done correctly, and that makes it difficultl. Decompiling their code and copying that would be illegal AFAIK.

**Comment by rbk0** [comment]
Never said I wanted to copy code and decompiling isn't illegal btw. Just copy code does not work in many cases neither.

I just asked, because someone possibly has access to documentation about the (high-level) protocol used. I'm not into Android dev, neither a full-time dev and try to get a way around reversing the protocol, because it takes a fair amount of time. And time is what I'm trying to save ;-)

**Comment by ashimokawa** [comment]
@rbk0
Bip uses a proprietary low level protocol.

**Comment by rbk0** [comment]
@ashimokawa 
Just realised now, that you're one of the main contributors. Thanks for the great work, you and the others contributed to this project!

I just skimmed through the code to find the implementation for other devices that use the GPS signal form the phone such as Mi Band 2.
Followed different locations (GPSCoordinate.java and others), but couldn't find anything.
Can you give me a hint where a similar GPS feature is implemented for another device? I guess there are some devices that already use this feature.
